Code/Framework/AzToolsFramework/Tests/Input/QtEventToAzInputMapperTests.cpp

```diff
@@ -533,23 +533,38 @@ namespace UnitTest
     {
     };
 
-    TEST_P(MoveMoveWrapParamQtEventToAzInputMapperFixture, MouseMove_NoAzHandlers_VerifyMouseMovmentViewport) 
+    TEST_P(MoveMoveWrapParamQtEventToAzInputMapperFixture, MouseMove_NoAzHandlers_VerifyMouseMovementViewport)
     {
+        //TODO: mouseMove is bugged mapToGlobal is called twice
+        auto mouseMoveFix = [](QWidget* wid, QPoint globalPos, QPoint deltaPos)
+        {
+            QPoint globalPosition = globalPos + deltaPos;
+            QPoint localPosition = wid->mapFromGlobal(globalPosition);
+            QTest::mouseMove(wid, localPosition);
+            QMouseEvent mouseMoveEvent(QEvent::MouseMove, localPosition, globalPosition, Qt::NoButton, Qt::NoButton, Qt::NoModifier);
+            QApplication::sendEvent(wid, &mouseMoveEvent);
+        };
```

Comment:
update: Apologies if I didn't mention this but we already have a workaround/fix for this in `AzToolsFrameworkHelpers.h` - see MouseMove and MousePressAndMove

```
// Note: There are a series of bugs in Qt that appear to be preventing mouseMove events
// firing when sent through the QTest framework. This is a work around for our version
// of Qt. In future this can hopefully be simplified. See ^1 for workaround.
// More info: Issues with mouse move in Qt
// - https://bugreports.qt.io/browse/QTBUG-5232
// - https://bugreports.qt.io/browse/QTBUG-69414
// - https://lists.qt-project.org/pipermail/development/2019-July/036873.html
void MouseMove(QWidget* widget, const QPoint& initialPositionWidget, const QPoint& mouseDelta, const Qt::MouseButton mouseButton)
{
    QPoint nextPosition = widget->mapToGlobal(initialPositionWidget + mouseDelta);

    // ^1 To ensure a mouse move event is fired we must call the test mouse move function
    // and also send a mouse move event that matches. Each on their own do not appear to
    // work - please see the links above for more context.
    QTest::mouseMove(widget, nextPosition);
    QMouseEvent mouseMoveEvent(
        QEvent::MouseMove, QPointF(nextPosition), QPointF(nextPosition), Qt::NoButton, mouseButton, Qt::NoModifier);
    QApplication::sendEvent(widget, &mouseMoveEvent);
}

Hopefully you can use this instead and remove the TODO comment
```

Reply:
I mean the mouse move in aztools is wrong. It now the cursor to global and passes it into qtest mouse move. Qtest mouse move expected the position to be local to the widget. I linked to the relevant qt code with how mouse move  works.

Reply:
Hmm okay I see, well the best thing to do in this case is create a new separate GHI with this bug, and include that in the code comment (e.g. `GHI-1234 - there is a bug with mouseMove where mapToGlobal is called twice, this is a temporary workaround`). Something like that?

Reply:
We try to avoiding adding 'TODOs' and always add a relevant ticket/id to track follow-up work